core/llm/index.ts

```diff
@@ -414,9 +415,11 @@ export abstract class BaseLLM implements ILLM {
             e.code === "ECONNREFUSED" &&
             e.message.includes("http://127.0.0.1:11434")
           ) {
-            throw new Error(
-              "Failed to connect to local Ollama instance, please ensure Ollama is both installed and running. You can download Ollama from https://ollama.ai.",
-            );
+            const message = (await isOllamaInstalled()) ?
+              "Failed to connect to local Ollama instance. Ollama appears to be stopped. It needs to be running." :
+              "Failed to connect to local Ollama instance, please ensure Ollama is both installed and running. You can download Ollama from https://ollama.ai."
```

Comment:
The notification text is a bit long here. Personally I'd drop the download link - it's too much detail for a notification and you'd hope that someone could figure out what to do.

```suggestion
              "Unable to connect to local Ollama instance. Ollama may not be installed or may not running."
```